### DISCUSSION TITLE
DB926972-EC77-714D-36BA-BA2244A7220A

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#207] [General]
# DB926972-EC77-714D-36BA-BA2244A7220A

This addon adds fictional textures for all planets of the 55 Cancri system. 55 Cancri, also known as Rho1 Cancri, was one of the first multi-planet systems discovered, and has a transiting carbon-rich super-Earth and four gas giants.

The previous version of this add-on has an issue, please uninstall and reinstall if you have an early version.

https://celestia.mobi/addon?item=DB926972-EC77-714D-36BA-BA2244A7220A

**Comment by SilverXplors** [comment]
One of my favorite exoplanets is in this addon.
